pkg/applications/github/on_tag_created.go

```diff
@@ -0,0 +1,193 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"slices"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/superplanehq/superplane/pkg/configuration"
+	"github.com/superplanehq/superplane/pkg/core"
+)
+
+type OnTagCreated struct{}
+
+type OnTagCreatedMetadata struct {
```

Comment:
We have a common struct - NodeMetadata - for node metadata, defined in common.go. We should use that instead.